Easydict/Swift/View/AcknowListView/AcknowListView.swift

```diff
@@ -23,7 +23,10 @@ struct AcknowListView: View {
     var body: some View {
         NavigationStack {
             AcknowListSwiftUIView(acknowList: acknowList)
+                .listStyle(.sidebar)
```

Comment:
Actually I think Form + formStyle(.grounded) would be more fitted in this scene.

But it turns out AcknowListSwiftUIView has hardcoded something. And the result is the header become one of the child element instead of a Form header.

<img width="830" alt="image" src="https://github.com/user-attachments/assets/2e75cf74-22bd-4c75-aa84-2d9c0c0d254f">